test/Xml/RequestHandlerTest.ts

```diff
@@ -336,7 +336,7 @@ describe("RequestHandler", () => {
             chai.assert.isOk(false, "Expected exception not thrown");
         } catch (ex) {
             chai.assert.instanceOf(ex, FetchError);
-            chai.assert.equal(ex.message, "524");
+            chai.assert.equal(ex.code, "524");
```

Comment:
why did this change from `message` to `code`?

Reply:
This was changed due to refactoring for FetchError changes in which it used to take 4 parameters and now takes in only 3. In this case, message was coming as empty and only the error code was required